pilot/pkg/bootstrap/server_test.go

```diff
@@ -112,6 +113,12 @@ func TestNewServer(t *testing.T) {
 			domain:         "mydomain.com",
 			expectedDomain: "mydomain.com",
 		},
+		{
+			name:           "override default secured grpc port",
+			domain:         "",
+			expectedDomain: constants.DefaultKubernetesDomain,
+			secureGRPCport: ":31128",
```

Comment:
I am not sure how we can verify the addr take effect from this test.

Reply:
you are correct, the testing will actually skip to create the securedDiscovery Server, So we will need a integration test instead of a UT.